PWGHF/TableProducer/treeCreatorOmegacSt.cxx

```diff
@@ -368,7 +416,8 @@ struct HfTreeCreatorOmegacSt {
   template <typename TracksType>
   void fillTable(Collisions const& collisions,
                  aod::AssignedTrackedCascades const& trackedCascades,
-                 aod::TrackAssoc const& trackIndices)
+                 aod::TrackAssoc const& trackIndices,
+                 std::optional<aod::McParticles> mcParticles = std::nullopt)
```

Comment:
AFAIK this will trigger an implicit type conversion `T -> std::optional<T>` which will result in a copy of the table.
See:
- https://www.think-cell.com/en/career/devblog/the-merits-of-optional-t-const
- https://abseil.io/tips/163